deebot_client/capabilities.py

```diff
@@ -189,6 +190,7 @@ class CapabilitySettings:
     )
     border_switch: CapabilitySetEnable[BorderSwitchEvent] | None = None
     child_lock: CapabilitySetEnable[ChildLockEvent] | None = None
+    cut_direction: CapabilitySet[CutDirectionEvent, int] | None = None
```

Comment:
I would create a MowerCapabilitySettings class instead, which extends this one. As a vacuum will never have a cut direction

Reply:
Thx for the pointer :-)

Reply:
Done, I've reverted the last commit.